package.json

```diff
@@ -52,7 +52,9 @@
 	},
 	"dependencies": {},
 	"devDependencies": {
+		"cldr-data": ">=26",
```

Comment:
While this package is pretty convenient, it also blows up an installed node_modules folder by 250 to 300 MB. @rxaviers is there any way to get an English-only cldr-data, since that's all we care about here (I think)?